graylog2-web-interface/src/views/components/aggregationwizard/__tests__/AggregationWizard.sort.test.tsx

```diff
@@ -201,6 +204,7 @@ describe('AggregationWizard', () => {
     const newSortContainer = await screen.findByTestId('sort-element-0');
     const applyButton = await findWidgetConfigFormSubmitButton();
     await waitFor(() => expect(within(newSortContainer).getByText('Field is required.')).toBeInTheDocument());
+    // eslint-disable-next-line jest/valid-expect
```

Comment:
We should not suppress the linter hint, but fix the next line, there is a double `expect`.